cli/src/opts/wallet.rs

```diff
@@ -209,17 +210,24 @@ pub trait WalletTrait {
 
     fn get_from_mnemonic(
         &self,
-        path: &str,
-        passphrase: Option<&str>,
-        derivation_path: Option<&str>,
+        mnemonic: &String,
+        passphrase: Option<&String>,
+        derivation_path: Option<&String>,
         index: u32,
     ) -> Result<LocalWallet> {
-        let mnemonic = fs::read_to_string(path)?.replace('\n', "");
+        let mnemonic = if Path::new(mnemonic).is_file() {
+            fs::read_to_string(mnemonic)?.replace('\n', "")
```

Comment:
I think this would be problematic on windows.

this should be equivalent:

```suggestion
            fs::read_to_string(mnemonic)?.lines().collect::<String>()
```